CMakeLists.txt

```diff
@@ -74,10 +74,13 @@ find_package(PythonInterp 2.7 REQUIRED)
 
 #by default, tall and skinny Gemm is enabled (for better performance)
 option(GEMM_TALL_SKINNY_SUPPORT "Whether to enable tall and skinny Gemm" ON)
+#By default vectorisation in gemm kernels is enabled for better performance
+option(GEMM_VECTORISATION_SUPPORT "Whether to enable vectorisation in Gemm kernels" OFF)
```

Comment:
if it is by default enabled the value shoud be ON

Reply:
Ah my mistake, forgot to update the comment. I am leaving it OFF by default for now until all the gemm kernels are vectorised and configurations optimised.